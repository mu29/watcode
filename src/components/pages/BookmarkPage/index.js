import React from 'react'
import { Header, GenericTemplate } from 'components'
import { BookmarkTab } from 'containers'

const BookmarkPage = () => (
  <GenericTemplate header={ <Header location="/bookmarks" /> }>
    <BookmarkTab />
  </GenericTemplate>
)

export default BookmarkPage
