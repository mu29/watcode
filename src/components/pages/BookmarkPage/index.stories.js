import React from 'react'
import { storiesOf } from '@storybook/react'
import BookmarkPage from '.'

storiesOf('Pages/BookmarkPage', module)
  .add('기본', () => (
    <BookmarkPage />
  ))
