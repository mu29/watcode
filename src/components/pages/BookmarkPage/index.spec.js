import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'
import BookmarkPage from '.'

const wrap = (props = {}) => shallow(<BookmarkPage { ...props } />)

describe('<BookmarkPage />', () => {
  it('정상적으로 렌더링된다', () => {
    const wrapper = wrap()
    expect(wrapper).toMatchSnapshot()
  })
})
