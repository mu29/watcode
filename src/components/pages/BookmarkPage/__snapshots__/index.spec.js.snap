// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BookmarkPage /> 정상적으로 렌더링된다 1`] = `
<GenericTemplate
  header={
    <Header
      location="/bookmarks"
    />
  }
>
  <StyledTab>
    <TabPage
      name="즐겨찾기"
    >
      <Connect(ArtworkListContainer) />
    </TabPage>
    <TabPage
      name="추천 작품"
    >
      <Message>
        준비중입니다 :)
      </Message>
    </TabPage>
  </StyledTab>
</GenericTemplate>
`;
